src/modules/fancyzones/FancyZonesLib/FancyZonesData.cpp

```diff
@@ -149,14 +152,34 @@ FancyZonesData& FancyZonesDataInstance()
 
 FancyZonesData::FancyZonesData()
 {
-    std::wstring saveFolderPath = PTSettingsHelper::get_module_save_folder_location(NonLocalizable::FancyZonesStr);
+    std::wstring saveFolderPath = PTSettingsHelper::get_module_save_folder_location(NonLocalizable::ModuleKey);
 
     settingsFileName = saveFolderPath + L"\\" + std::wstring(NonLocalizable::FancyZonesSettingsFile);
     zonesSettingsFileName = saveFolderPath + L"\\" + std::wstring(NonLocalizable::FancyZonesDataFile);
     appZoneHistoryFileName = saveFolderPath + L"\\" + std::wstring(NonLocalizable::FancyZonesAppZoneHistoryFile);
     editorParametersFileName = saveFolderPath + L"\\" + std::wstring(NonLocalizable::FancyZonesEditorParametersFile);
 }
 
+void FancyZonesData::ReplaceZoneSettingsFileFromOlderVersions()
+{
+    if (std::filesystem::exists(zonesSettingsFileName))
```

Comment:
once it's migrated, no need to do it every time? add ` && if NOT exist hotkeysFile` to condition?

Reply:
I'm planning to add the code removing the `zones-settings.json` after I finish with other parts. So checking only this one will be enough.

Reply:
ok. that makes sense